### PR TITLE
Update Set-Acl.md

### PR DESCRIPTION
**Pull request by JarrettRenshaw**
PLease contact Herbert Mauerer <EUROPE\herbertm> with any questions.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C12%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201612131701464694-901%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by juanpablojofre** [comment]
@JarrettRenshaw , could you please update the other versions (3.0, 4.0, 5.0) of the cmdlet or state why this update doesn't apply to earlier versions of the cmdlet.
Once done, we'll merge your PR.
Thanks a lot for your contribution.

**Comment by JarrettRenshaw** [comment]
Hi JuanPablo, we only received the complaint about this version, the requestor is not sure if it applies to other versions.

Regards,

Jarrett

**Comment by JarrettRenshaw** [comment]
Hi, I have responded above, can this request be merged? Thanks.

**Comment by JarrettRenshaw** [comment]
Hi all, I don't see any continuation of this since 3/22. I understand this applies to other cmdlets, but can we update them as the need arises (mainly from CSS needs to explain to customers)?

File: reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md [hunk 1]
```diff
@@ -201,8 +203,8 @@ Accept wildcard characters: False
 Specifies a filter in the provider's format or language.
 The value of this parameter qualifies the *Path* parameter.
 The syntax of the filter, including the use of wildcards, depends on the provider.
-Filters are more efficient than other parameters, because the provider applies them when retrieving the objects, rather than having Windows PowerShell filter the objects after they are retrieved.
-
+Filters are more efficient than other parameters, because the provider applies them when retrieving the objects, rather than having Windows PowerShell filter the objects after they are retrieved.
```
Comment (joeyaiello) [inline reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md]: This trailing space shouldn't have been added.

**Comment by sdwheeler** [comment]
@zjalexander Per our conversation, I am closing this PR unmerged. The changes are being implemented in PR #1844

**Pull request closed without merging**